src/containers/SongRow/SongRow.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react';
+import './SongRow.css'
+
+class SongRow extends Component {
+  constructor(props) {
+    super(props);
+    if (! props.song.votes.length) {
+      props.song.votes = [{score: 0}];
+    }
+
+    this.state = {
+      song: props.song,
+      songIndex: props.songIndex
+    }
+
+    this.votePlus = this.votePlus.bind(this);
+    this.voteMin = this.voteMin.bind(this);
+  }
+
+  votePlus() {
+    const song = this.state.song;
+    song.votes[0].score++;
```

Comment:
Never mutate this.state directly, as calling setState() afterwards may replace the mutation you made. Treat this.state as if it were immutable.